android/settings.gradle

```diff
@@ -1,4 +1,24 @@
 rootProject.name = 'LNDR'
+include ':urbanairship-react-native'
+project(':urbanairship-react-native').projectDir = new File(rootProject.projectDir, '../node_modules/urbanairship-react-native/android')
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
+include ':react-native-touch-id'
+project(':react-native-touch-id').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-touch-id/android')
+include ':react-native-splash-screen'
+project(':react-native-splash-screen').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-splash-screen/android')
+include ':react-native-randombytes'
+project(':react-native-randombytes').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-randombytes/android')
+include ':react-native-image-resizer'
+project(':react-native-image-resizer').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-image-resizer/android')
+include ':react-native-image-picker'
+project(':react-native-image-picker').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-image-picker/android')
+include ':react-native-firebase'
+project(':react-native-firebase').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-firebase/android')
+include ':react-native-fetch-blob'
+project(':react-native-fetch-blob').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-fetch-blob/android')
+include ':react-native-blur'
+project(':react-native-blur').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-blur/android')
```

Comment:
Whoa, lots of new projects added in this.

:stars: Prior to Gradle, Android projects were primarily built using Ant, while some more sophisticated projects were using Maven. Maven is to Ant what NPM is to Gulp. For whatever reason, the Android ecosystem never quite settled on Maven, despite its obvious superiority to Ant in a project with dependencies. When Gradle came along, though, the developers who had been using Maven were like, "Hmm... this looks very familiar." That's because Gradle is *heavily* inspired by Maven. You see, Ant-based Android projects needed a ton of boilerplate. Maven projects still needed some boilerplate, but Gradle... Gradle was supposed to make boilerplate a thing of the past. "Convention over configuration" philosophy.

All of this to say... in a project using Gradle, if there's a pile of copy/paste code or boilerplate, there's probably something wrong with how we're doing things.

Reply:
most of these verbose import sections are generated by commands like `react-native link`